aptos-move/framework/aptos-token/sources/PropertyMap.move

```diff
@@ -1,6 +1,6 @@
 /// PropertyMap is a specialization of SimpleMap for Tokens.
 /// It maps a String key to a PropertyValue that consists of a String type and a vec value
-module AptosFramework::PropertyMap {
+module AptosToken::PropertyMap {
```

Comment:
shorthands can be very confusing. If "vec" means vector, then I suggest change it to "vector". That way the reader can confirm it clearly by looking at the code.

Reply:
updated